netplan/cli/commands/apply.py

```diff
@@ -214,7 +214,9 @@ def command_apply(self, run_generate=True, sync=False, exit_on_error=True):  # p
         if restart_networkd:
             netplan_wpa = [os.path.basename(f) for f in glob.glob('/run/systemd/system/*.wants/netplan-wpa-*.service')]
             netplan_ovs = [os.path.basename(f) for f in glob.glob('/run/systemd/system/*.wants/netplan-ovs-*.service')]
-            utils.systemctl_networkd('start', sync=sync, extra_services=netplan_wpa + netplan_ovs)
+            # Run 'systemctl start' command synchronously, to avoid race conditions
+            # with 'oneshot' systemd service units, e.g. netplan-ovs-*.service.
+            utils.systemctl_networkd('start', sync=True, extra_services=netplan_wpa + netplan_ovs)
```

Comment:
I think this now makes sense. Doing this asynchronously in the past was something we could do, but now we have much much more services that can potentially cause issues. +1